pre-docs/bolt_configuration_options.md

```diff
@@ -40,6 +40,8 @@ interpreters:
 
 `inventoryfile`: The path to a structured data inventory file used to refer to groups of nodes on the commandline and from plans. The default path for the inventory file is `inventory.yaml` inside the Bolt project directory.
 
+`puppetfile`: A map containing options for the `bolt puppetfile install` command. The only allowed key is `proxy` which specifies the HTTP proxy to use for Git and Puppet Forge operations.
```

Comment:
Do we want to constrain what we accept here or just rely on documentation.

Reply:
I updated it to only pull out the supported key.